Dockerfile

```diff
@@ -85,21 +93,45 @@ RUN apt-get install gawk && \
 
 RUN apt-get install -y libssl-dev libsqlite3-dev libbz2-dev
 
-WORKDIR /tmp
 
-# Add python 3.6
-RUN wget https://www.python.org/ftp/python/3.6.6/Python-3.6.6.tgz && \
-    tar xvf Python-3.6.6.tgz && \
-    cd Python-3.6.6 && \
-    ./configure --enable-optimizations --enable-loadable-sqlite-extensions && \
-    make -j8 build_all && \
-    make altinstall
+########################## ESPNET INSTALLATION #########################
+
+# Some ESPnet dependencies may be covered above but listing all for the sake of completeness
+RUN echo "===> Install ESPnet dependencies" && \
+    apt-get update && apt-get install -y cmake \
+    sox \
+    ffmpeg \
+    flac \
+    bc
+
+WORKDIR /
+
+# Setting up ESPnet for Elpis from Persephone repository
+RUN git clone https://github.com/persephone-tools/espnet.git
+
+WORKDIR /espnet
+
+RUN git checkout elpis
+
+# Explicitly installing only the CPU version. We should update this to be an
+# nvidia-docker image and install GPU-supported version of ESPnet.
+WORKDIR /espnet/tools
+
+RUN echo "===> install ESPnet" && \
+    make KALDI=/kaldi CUPY_VERSION='' -j 4
 
-# Add python packages and their dependencies
-RUN apt-get install -y python3-dev python3-pip python3-certifi python3-venv
+
+########################## DEV HELPERS INSTALLATION ####################
+
+WORKDIR /tmp
 
 # Add a task runner
-RUN curl -s https://taskfile.org/install.sh | sh && mv ./bin/task /bin/
+#RUN curl -s https://taskfile.org/install.sh | sh && mv ./bin/task /bin/
```

Comment:
If this isn't necessary it should be removed.

Reply:
will remove